packages/backend/src/core/activity/counters/AztecConnectCounter.ts

```diff
@@ -49,7 +46,9 @@ export function createAztecConnectCounter(
           }
         })
 
-        const blocks = await promiseAllPlus(queries, logger)
+        const blocks = await promiseAllPlus(queries, logger, {
+          metricsId: `AztecConnectBlockCounter[${projectId.toString()}]`,
```

Comment:
There's only one AztecConnect ;)
```suggestion
          metricsId: 'AztecConnectBlockCounter',
```